cmd/internal/ascii/cmd.go

```diff
@@ -0,0 +1,24 @@
+package ascii
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:          "aki",
+	Short:        "Show an ASCII Akita",
+	Long:         "Show an ASCII Akita",
+	SilenceUsage: true,
+	Hidden:       true,
+	RunE:         showAki,
+}
+
+var aki string = " ___                 ___\n >_ \\   _ _ _ _ _   / _<\n< <, Y``         ''Y .> >\n \\ (`  .-.-._.-.-.  ') /\n ( ` .'  _     _  `. ' )\n )  (   (')___(')   )  ( \n(  ('      `Y'      `)  )\n(   (.  `.__|__.'  .)   )\n (.  (._         _.)  .)\n  (_   `- - - - -'   _)\n    `-._ _ ___ _ _.-´  \n"
```

Comment:
Feels like this belongs in an embedded file, viz. go:embed. Would make it easier to include ANSI codes too.